src/pages/HomePage/HomeInfo.js

```diff
@@ -27,7 +28,7 @@ const HomeInfo = () => {
               <FormattedMessage id="app.welcome.item2" />
             </p>
           </div>
-          <button type="button" className="button">
+          <button type="button" className="button" onClick={() => history.push('/')}>
```

Comment:
How about using a [Link](https://reactrouter.com/web/guides/primary-components/navigation-or-route-changers) component instead? It's always good to use the native <a> elements for navigation instead of programmatically navigating from JS